krod/core/llm_manager.py

```
@@ -5,11 +5,12 @@
 import logging
 import time
 import os
+import json
+import openai
```

Ruff (0.11.9)

`openai` imported but unused

Remove unused import: `openai`

(F401)